packages/experiments/src/index.js

```diff
@@ -27,6 +27,12 @@ const registeredExperiments = {};
 const requiredConsent =
 	'I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.';
 
+const privateAPIs = new WeakMap();
+export const experimentsSymbol = Symbol( 'experiments' );
```

Comment:
I didn't commit all the changes to this file, this is missing:

```suggestion
export const experimentsSymbol = Symbol( 'experiments' );
export const experimentalAPIsDecorator = Symbol( 'experimentalAPIsDecorator' )
```